bdb/bdb_access.c

```diff
@@ -276,6 +276,15 @@ static int bdb_del_user_tbl_access(bdb_state_type *bdb_state, tran_type *tran,
     int rc = 0;
     int bdberr;
 
+     if ((rc = bdb_tbl_op_access_delete(bdb_state, tran, 0, table_name,
```

Comment:
Looks good to me! Think the layout is one too many spaces here than from the rest of the file though

Reply:
Fixed, thanks.